src/sentry/migrations/0639_index_together.py

```diff
@@ -0,0 +1,267 @@
+# Generated by Django 4.2.8 on 2024-01-29 10:55
+
+from django.db import migrations
+
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+class Migration(CheckedMigration):
+    # This flag is used to mark that a migration shouldn't be automatically run in production. For
+    # the most part, this should only be used for operations where it's safe to run the migration
+    # after your code has deployed. So this should not be used for most operations that alter the
+    # schema of a table.
+    # Here are some things that make sense to mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that they can
+    #   be monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
+    #   have ops run this and not block the deploy. Note that while adding an index is a schema
+    #   change, it's completely safe to run the operation after the code has deployed.
+    is_dangerous = False
+
+    dependencies = [
+        ("sentry", "0638_add_date_added_to_dashboard_widget_on_demand"),
+    ]
+
+    operations = [
```

Comment:
What was the reason to rename the indexes ?

Reply:
Django is just doing it because we changed how we're defining the indexes. It's not required those imo - we could just mark this migration as dangerous and never run it if it's a huge pain.

Reply:
the change is uncovering a lot of missed past dangerous migrations -- I think getting those applied is probably for the best anyway so things are more consistent 🤷